src/shared-components/button/components/roundButton/style.ts

```diff
@@ -54,14 +54,14 @@ export const RoundButtonBase = styled(ButtonBase)`
   }
 `;
 
-export const roundButtonLoader = (disabled: boolean) => css`
+export const roundButtonLoader = (disabled: boolean, theme: ThemeType) => css`
   width: 36px;
   margin: -3px -3px 0 0;
 
   ${disabled &&
```

Comment:
good catch here.  Aren't  we were replacing all `&&` with ternary operator

Reply:
It's only necessary when in a `styled` tagged template literal. `css` cleans up `false` statements but `jest` tests would fail from CSS parse issues for the same in `styled` usage.